diff -Naur telldus-core-2.1.1/common/Socket.h telldus-core-2.1.1.patch/common/Socket.h
--- telldus-core-2.1.1/common/Socket.h	2012-02-03 16:18:15.000000000 +0100
+++ telldus-core-2.1.1.patch/common/Socket.h	2013-12-30 08:09:32.623193785 +0100
@@ -7,6 +7,7 @@
 	#include <windows.h>
 	typedef HANDLE SOCKET_T;
 #else
+        #include <unistd.h>
 	typedef int SOCKET_T;
 #endif
 
diff -Naur telldus-core-2.1.1/service/Log.h telldus-core-2.1.1.patch/service/Log.h
--- telldus-core-2.1.1/service/Log.h	2012-02-03 16:18:15.000000000 +0100
+++ telldus-core-2.1.1.patch/service/Log.h	2013-12-30 08:54:40.627263123 +0100
@@ -2,6 +2,7 @@
 #define LOG_H
 
 #include <stdio.h>
+#include <stdarg.h>
 
 class Log {
 public:
